sky/spot/spot_state.py

```diff
@@ -99,7 +101,7 @@ def set_submitted(job_id: int, name: str, run_timestamp: str,
         status=(?),
         run_timestamp=(?)
         WHERE job_id=(?)""",
-        (name, resources_str, time.time(), SpotStatus.SUBMITTED.value,
+        (name, resources_str, submit_time, SpotStatus.SUBMITTED.value,
```

Comment:
Hmm, what's the reason to use this earlier time vs. `time.time()`?  

This rings a bell because the table schema has `submitted_at` and `run_timestamp`. Now we're making both columns store the same time.

Reply:
The variable name `run_timestamp` is a bit misleading, whose actual meaning is the run_id for the run `sky-2022-10-04....`, i.e. the logs will be stored in f`~/sky_logs/{run_timestamp}`.

Here I want to make the timestamp for submission, sky_logs and job id align with each other, so as to make it easier to find them based on one of the values. 

Also, using the earlier timestamp should be closer to the term `submit_at`, which should represent the time the spot task is submitted.

Reply:
Added the comment. Thanks!